betty/extension/webpack/webpack/webpack.config.js

```diff
@@ -0,0 +1,178 @@
+'use strict'
+
+import { CleanWebpackPlugin } from 'clean-webpack-plugin'
+import CssMinimizerPlugin from 'css-minimizer-webpack-plugin'
+import MiniCssExtractPlugin from 'mini-css-extract-plugin'
+import path from 'path'
+import { readFile } from 'node:fs/promises'
+import TerserPlugin from 'terser-webpack-plugin'
+import url from 'node:url'
+import webpack from 'webpack'
+
+const __dirname = url.fileURLToPath(new URL('.', import.meta.url))
+const configuration = JSON.parse(await readFile('./webpack.config.json'))
+
+/**
+ * Collect the scripts needed for all entrypoints, and build a loader.
```

Comment:
Explain WHY we use this roundabout way of loading entrypoints: because we build them separately from the individual pages and we don't know what each page needs until it is done rendering